src/query/config/src/inner.rs

```diff
@@ -170,6 +170,7 @@ pub struct QueryConfig {
     /// If in management mode, only can do some meta level operations(database/table/user/stage etc.) with metasrv.
     pub management_mode: bool,
     pub jwt_key_file: String,
+    pub additional_jwt_key_files: Vec<String>,
```

Comment:
how about just calls it `jwt_key_files`? in most time we only need one of `jwt_key_file` or `jwt_key_files`.